NEWS.md

```diff
@@ -18,6 +18,8 @@
 * Added support for setting the `subPath` on Kubernetes sessions using `KubernetesPersistentVolumeClaim` mounts in `/etc/rstudio/launcher-mounts` (Pro #2976).
 * Fixed custom shortcuts not appearing correctly in menus (#9915)
 * Fixed custom shortcuts not appearing correctly in "Keyboard Shortcuts Help" and Electron menus. (#9953)
+* File pane controls adjust based on the width of the pane so they are always visible (#9870)
```

Comment:
```suggestion
```
Looks like you accidentally kept the original line and added a new one for the reword